tests/system/lab/verify_clean_pi.py

```diff
@@ -34,7 +36,7 @@
 pi_db = args["pi_db"]
 asset_name = args["asset_name"]
 
-af_hierarchy_level = "PIlabSinelvl1/PIlabSinelvl2/PIlabSinelvl3"
+af_hierarchy_level = "{0}_{1}_{2}_PIlabSinelvl1/PIlabSinelvl2/PIlabSinelvl3".format(today.strftime("%Y"), today.strftime("%m"), today.strftime("%d"))
```

Comment:
`today = datetime.now().strftime("%Y_%m_%d")` and use `"{}_PI...".format(today)`